x-pack/test/idp-fixture/build.gradle

```diff
@@ -7,3 +11,26 @@ dependencies {
   api project(':test:fixtures:testcontainer-utils')
   api "junit:junit:${versions.junit}"
 }
+
+tasks.register("deployIdpFixtureDockerImages", DockerBuildTask) {
+    dockerContext.fileValue(file("src/main/resources/idp"))
+    baseImages = ["openjdk:11.0.16-jre"]
+    noCache = BuildParams.isCi
+    tags = ["docker.elastic.co/elasticsearch-dev/idp-fixture:1.0"]
+    getPush().set(BuildParams.isCi)
+    getPlatforms().addAll( Architecture.values().collect{ it.dockerPlatform } )
+}
+
+
+tasks.register("deployOpenLdapFixtureDockerImages", DockerBuildTask) {
+  dockerContext.fileValue(file("src/main/resources/openldap"))
+  baseImages = ["osixia/openldap:1.4.0"]
+  noCache = BuildParams.isCi
+  tags = ["docker.elastic.co/elasticsearch-dev/openldap-fixture:1.0"]
+  getPush().set(true)
+  getPlatforms().addAll( Architecture.values().collect{ it.dockerPlatform } )
+}
+
+tasks.register("deployFixtureDockerImages") {
+  dependsOn tasks.withType(DockerBuildTask)
+}
```

Comment:
We can move parts of this to an internal plugin once we do this fixture pre baking for more fixtures. We can evaluate existing fixtures in a later pr

Reply:
Sounds good.